springbackend/src/main/java/com/simplecoding/social/service/FriendService.java

```diff
@@ -64,11 +76,15 @@ public void saveFriend(UserDto userDto1, int id) throws NullPointerException{
         UserDto currentUserDto = securityService.getUser();
         User currentUser = userRepository.findUserByEmail(currentUserDto.getEmail());
         List<Friend> friends = friendRepository.findByFirstUser(currentUser);
-        List<User> friendUsers = new ArrayList<User>();
+        List<Friend> friends1 = friendRepository.findBySecondUser(currentUser);
```

Comment:
if you lexiographically adding when do you need to get friends by both first and second user?